sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor.result

```diff
@@ -0,0 +1,140 @@
+===================================================
```

Comment:
Is this file required?

Reply:
sorry, this file is not needed.
I'm removed from this.